projects/github-actions/repo-gardening/src/utils/slack/send-slack-message.js

```diff
@@ -77,18 +79,71 @@ async function sendSlackMessage( message, channel, payload, customMessageFormat
 		};
 	}
 
-	const slackRequest = await fetch( 'https://slack.com/api/chat.postMessage', {
-		method: 'POST',
-		body: JSON.stringify( slackMessage ),
-		headers: {
-			'Content-Type': 'application/json; charset=utf-8',
-			'Content-Length': slackMessage.length,
-			Authorization: `Bearer ${ token }`,
-			Accept: 'application/json',
-		},
-	} );
+	try {
+		const slackRequest = await slackApi.chat.postMessage( slackMessage );
+		return !! slackRequest.ok;
+	} catch ( error ) {
+		// The request failed.
+		// At this point, we want to log specific types of errors (let's avoid noise by logging temporary errors for example).
+		if ( ! error.code === ErrorCode.PlatformError ) {
+			return false;
+		}
+
+		// See the list of error messages here: https://api.slack.com/methods/chat.postMessage#errors
+		const errorMessage = error?.data?.error ?? 'Unknown error';
+
+		// Let's send a direct message to @jeherve about it, so we can investigate.
+		// For folks outside of Automattic, let's use the Quality team channel.
```

Comment:
Why not send all to the channel?

Reply:
I think that's what we'll end up doing on the long term, but for now I'd like to monitor the assignment list closely, especially as more teams get re-orged.